codeHF/config_tasks.sh

```diff
@@ -384,6 +386,8 @@ function MakeScriptO2 {
   [ $DOO2_DPLUSDMINUS_DATA -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-dplus-dminus o2-analysis-hf-task-correlation-d-dbar"
   [ $DOO2_DPLUSDMINUS_MCREC -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-dplus-dminus-mc-rec o2-analysis-hf-task-correlation-d-dbar-mc-rec"
   [ $DOO2_DPLUSDMINUS_MCGEN -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-dplus-dminus-mc-gen o2-analysis-hf-task-correlation-d-dbar-mc-gen"
+  [ $DOO2_CORRELATOR_D0HADRON -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-d0-hadrons"
+  [ $DOO2_TASK_D0HADRON -eq 1 ] && WF_CORR="o2-analysis-hf-task-correlation-d0-hadrons"
   [ "$WF_CORR" ] && WORKFLOWS+=" $WF_CORR"
```

Comment:
```suggestion
  [ "$WF_CORR" ] && WORKFLOWS+=" $WF_CORR"
  [ $DOO2_CORRELATOR_D0HADRON -eq 1 ] && WORKFLOWS+=" o2-analysis-hf-correlator-d0-hadrons"
  [ $DOO2_TASK_D0HADRON -eq 1 ] && WORKFLOWS+=" o2-analysis-hf-task-correlation-d0-hadrons"
```
If I understand correctly, your code is independent of `o2-analysis-hf-task-correlation-d-dbar` and therefore also its activation should not interfere with it.